lib/knapsack_pro/config/ci/base.rb

```diff
@@ -32,6 +32,17 @@ def detected
         def fixed_queue_split
           true
         end
+
+        def ci_provider
+          return 'AWS CodeBuild' if ENV.key?('CODEBUILD_BUILD_ARN')
+          return 'Azure Pipelines' if ENV.key?('SYSTEM_TEAMFOUNDATIONCOLLECTIONURI')
+          return 'Bitbucket Pipelines' if ENV.key?('BITBUCKET_COMMIT')
+          return 'Google Cloud Build' if ENV.key?('BUILDER_OUTPUT')
+          return 'Jenkins' if ENV.key?('JENKINS_URL')
+          return 'TeamCity' if ENV.key?('TEAMCITY_VERSION')
+
```

Comment:
We could consider detecting also Buddy Works, Drone, Bamboo. I recall some users were using it.

Reply:
https://github.com/KnapsackPro/knapsack_pro-ruby/pull/216/commits/f05d807e7b772eaeb01d99ddb7bca44810d9442b